resources/views/auth/login.blade.php

```diff
@@ -1,85 +1,73 @@
 
-          <div class="card-body">
-            {{ Form::open(['route' => 'login']) }}
-              <div class="form-group row">
-                {{ Form::label('email', __('E-Mail Address'), [
-                  'class' => 'col-md-4 col-form-label text-md-right'
-                ]) }}
+                <div class="card-body">
+                    <form method="POST" action="{{ route('login') }}">
+                        @csrf
 
-                <div class="col-md-6">
-                  {{ Form::email('email', null, [
-                    'id' => 'email',
-                    'class' => 'form-control'.($errors->has('email') ? ' is-invalid' : ''),
-                    'required' => true,
-                    'autofocus' => true,
-                  ]) }}
+                        <div class="row mb-3">
+                            <label for="email" class="col-md-4 col-form-label text-md-end">{{ __('Email Address') }}</label>
 
-                  @error('email')
-                    <span class="invalid-feedback" role="alert">
-                      <strong>{{ $message }}</strong>
-                    </span>
-                  @enderror
-                </div>
-              </div>
+                            <div class="col-md-6">
+                                <input id="email" type="email" class="form-control @error('email') is-invalid @enderror" name="email" value="{{ old('email') }}" required autocomplete="email" autofocus>
 
-              <div class="form-group row">
-                {{ Form::label('password', __('Password'), [
-                  'class' => 'col-md-4 col-form-label text-md-right'
-                ]) }}
+                                @error('email')
+                                    <span class="invalid-feedback" role="alert">
+                                        <strong>{{ $message }}</strong>
+                                    </span>
+                                @enderror
+                            </div>
+                        </div>
 
-                <div class="col-md-6">
-                  {{ Form::password('password', [
-                    'id' => 'password',
-                    'class' => 'form-control'.($errors->has('password') ? ' is-invalid' : ''),
-                    'required' => true,
-                  ]) }}
+                        <div class="row mb-3">
+                            <label for="password" class="col-md-4 col-form-label text-md-end">{{ __('Password') }}</label>
 
-                  @error('password')
-                    <span class="invalid-feedback" role="alert">
-                      <strong>{{ $message }}</strong>
-                    </span>
-                  @enderror
-                </div>
-              </div>
+                            <div class="col-md-6">
+                                <input id="password" type="password" class="form-control @error('password') is-invalid @enderror" name="password" required autocomplete="current-password">
```

Comment:
Just a nitpick, I think it's clearer to have conditional classes inside `class` but leave it the way you better see fit

```suggestion
                                <input id="password" type="password"
                                 @class(['form-control', 'is-invalid' => $errors->has('password')])
                                 name="password" required autocomplete="current-password">
```